internal/pkg/bootstrap/bootstrap.go

```
@@ -127,7 +129,18 @@ func Bootstrap(cfg Config, logger log.Logger, file string) error {
 }
 
 func getConfig(bsID, bsKey, bsSvrURL string) (deviceConfig, error) {
-	client := &http.Client{}
+	// Get the SystemCertPool, continue with an empty pool on error
+	rootCAs, _ := x509.SystemCertPool()
```

Is this making HTTPS mandatory? It should be optional, selected by ENV vars

---

nope, it works with http as well

---

Please don't ignore the error here.